salt/modules/win_servermanager.py

```diff
@@ -104,7 +123,10 @@ def list_installed():
           '-ErrorAction SilentlyContinue ' \
           '-WarningAction SilentlyContinue ' \
           '| Select DisplayName,Name,Installed'
-    features = _pshell_json(cmd)
+    try:
+        features = _pshell_json(cmd)
+    except CommandExecutionError:
```

Comment:
Can you explain the purpose of this? I don't know why catching and then raising the same exception would be needed here.

Reply:
del is supposed to be faster than pop... and I don't need to know the deleted value